pages/dkp/konvoy/2.2/download/index.md

```diff
@@ -20,17 +20,11 @@ To download a new version of DKP, you have 2 options:
 You must be a registered user and logged on to the support portal to download DKP. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install this product.
 If you have problems downloading DKP, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.</p>
 
-## Download from the command line
 
-You can download image bundle files by running the following CLI command:
 
-    ```bash
-    wget "https://downloads.d2iq.com/dkp/v2.2.0/kommander-image-bundle-v2.2.0.tar.gz" -O kommander-image-bundle.tar.gz
-    ```
+## Download image from the Infrastructure Provider of choice marketplace
```

Comment:
This was a good idea, but the only marketplace we are currently in is AWS, so this needs to stay what it was before.   Maybe in the future...

```suggestion
## Download from the AWS Marketplace
```